### PR TITLE
[#2] Create empty jetpack compose project

### LINKED ISSUE #2: [Chore] Create android project with compose
## Why
Create android project with Jetpack Compose

## Acceptance Criteria
- Create new android project with jetpack compose and push it to Github
- Application must be able to build and run

### PR DESCRIPTION
**Pull request by kaungkhantsoe**
Closes [#2](https://github.com/kaungkhantsoe/NimbleSurveyJetpackCompose/issues/2)

## What happened 👀

Create android project with Jetpack Compose

## Insight 📝

- Create empty android project with jetpack compose

## Proof Of Work 📹

https://user-images.githubusercontent.com/32578035/182760816-da9c48e0-9efd-4ff8-863d-aba2bb7527ba.mov

### CODE REVIEW

File: app/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (ryan-conway) [inline app/.gitignore]: Missing newline here 👀

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (ryan-conway) [inline .idea/.gitignore]: I believe we generally don't checkin `.idea/` files into VCS 🤔 @AVI5HEK thoughts?

File: app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.3.1'
+    implementation 'androidx.activity:activity-compose:1.3.1'
+    testImplementation 'junit:junit:4.13.2'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+    androidTestImplementation "androidx.compose.ui:ui-test-junit4:$compose_version"
+    debugImplementation "androidx.compose.ui:ui-tooling:$compose_version"
+    debugImplementation "androidx.compose.ui:ui-test-manifest:$compose_version"
+}
```
Comment (ryan-conway) [inline app/build.gradle]: Another missing newline 👀

**Review by ryan-conway — COMMENTED** [review]
Looks like a majority of the files are missing a newline at eof 🤔 
The rest LGTM 👍🏻

File: app/.gitignore (hunk 1 above)
Comment (ryan-conway) [inline app/.gitignore]: missing newline here too 😅

**Comment by ryan-conway** [comment]
@kaungkhantsoe There are still many files that have a missing a newline at eof, you can see which ones by going through the [files tab](https://github.com/kaungkhantsoe/NimbleSurveyJetpackCompose/pull/55/files) on this PR and looking for the 🚫  symbol 😅 
Screenshot 2565-08-09 at 10 15 33

**Comment by kaungkhantsoe** [comment]
> @kaungkhantsoe There are still many files that have a missing a newline at eof, you can see which ones by going through the [files tab](https://github.com/kaungkhantsoe/NimbleSurveyJetpackCompose/pull/55/files) on this PR and looking for the 🚫 symbol 😅 Screenshot 2565-08-09 at 10 15 33

Update @ryan-conway 🙌

**Review by ryan-conway — APPROVED** [review]
LGTM now 👍🏻

File: .idea/.gitignore (hunk 2 above)
Comment (AVI5HEK) [reply .idea/.gitignore]: True. Was it added deliberately or auto-created? 🤔 

The root-level gitignore should be enough for the most part. [This](https://github.com/github/gitignore/blob/main/Android.gitignore) template contains the basic setup needed for an Android project. @kaungkhantsoe

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Note: for a release PR, append this parameter `?template=release_template.md` to the current URL to apply the release PR template, e.g. `{Github PR URL}?template=release_template.md`
+
+--
+
+https://github.com/nimblehq/android-templates/issues/??
```
Comment (AVI5HEK) [inline .github/PULL_REQUEST_TEMPLATE.md]: We can update it like this:
```suggestion
Closes #
```

File: app/build.gradle [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+            useSupportLibrary true
+        }
+    }
+
+    buildTypes {
+        debug {
+            signingConfig signingConfigs.debug
+        }
+        release {
+            minifyEnabled false
```
Comment (AVI5HEK) [inline app/build.gradle]: We should set `minifyEnabled` and `shrinkResources` to `true` and `debuggable` to `false` for `release`. But we can do it later if you have a plan for it.

**Review by AVI5HEK — COMMENTED** [review]
Overall looks good to me. How about adding a `CODEOWNERS` file to automatically assign the reviewers when creating PRs?

File: app/build.gradle (hunk 5 above)
Comment (kaungkhantsoe) [reply app/build.gradle]: Updated @AVI5HEK

**Review by AVI5HEK — APPROVED** [review]
LGTM 🚀

**Pull request merged**